modules/ROOT/pages/event-tracking.adoc

```diff
@@ -116,29 +123,33 @@ services:
  globalTrackingLevel: TRACKING
 ----
 
-[NOTE]
-If `globalTrackingLevel` is not specified, the default value is `NONE`.
+If you don't specify `globalTrackingLevel`, the default value is `NONE`.
 
-==== Configuring Tracking per Mule Application
+==== Configure Tracking for a Mule App
 
-In addition to setting a default global tracking level for every application, you can override the tracking levels per Mule application.
+You can override the default global tracking level by setting tracking levels per Mule app.
 
-There are three Mule application level tracking areas: trackingLevel, replayLevel, and trackedFlows.
+The three Mule app level tracking areas are: `trackingLevel`, `replayLevel`, and `trackedFlows`.
 
-* `trackingLevel` specifies one of the four tracking levels (`NONE`, `BE`, `TRACKING`, `DEBUG`) described in the previous table. This tracking level is applied to every flow in the Mule application.
-* `replayLevel` specifies if and how each message through every flow of the Mule application should be stored for possible later replay. The possible values are: `NONE`, `SOURCED`, or `ALL`.
+* `trackingLevel`
++
+Specifies one of the four tracking levels: `NONE`, `BE`, `TRACKING`, or `DEBUG`. This tracking level affects every flow in the Mule app. See <<Event Tracking Levels>>.
+* `replayLevel`'
++
+Specifies how to store each message through every flow of the Mule app for possible later replay: `NONE`, `SOURCED`, or `ALL`.
 
+[[Replay]]
 [%header%autowidth.spread]
 |===
 |Replay Level Option | Description
-|`NONE` |  Do not store any data related to message replay and disable the replay feature for these flows.
-|`SOURCED` | Only stores messages for later replay for each flow in the Mule application that starts with a message source.
-|`ALL` |  Stores messages for later replay for every flow in the Mule application.
+|`NONE` |  Specifies to not store any data related to message replay and to disable the replay feature for these flows.
+|`SOURCED` | Stores messages for later replay for only those flows in the Mule app that start with a message source.
+|`ALL` |  Stores messages for later replay for every flow in the Mule app.
 |===
 
-In the following example, the tracking level for the application `mule-flights` and its flow is set to `TRACKING`, and the message replay facility is disabled for this Mule application. The `mule-flights` application will send some notifications to the Event Tracking Service, but will not copy each received message, and none of the flows in the `mule-flights` application can be replayed from the Event Tracking Service.
+This example sets the tracking level for the app `mule-flights` and its flow to `TRACKING`, and disables the message replay facility for this Mule app. The `mule-flights` app sends some notifications to the Event Tracking Service, but doesn't copy each received message. None of the flows in the `mule-flights` app can be replayed from the Event Tracking Service.
```

Comment:
Consider omitting "for this Mule app."

ditto ? re. capitalization for Event Tracking Service